### ISSUE TITLE
unable to see any notes

### ISSUE THREAD

**Issue by awesomescot-zz** [FMCorz/moodle-repository_evernote#3]
Hi.  I've installed this plugin and setup on the sandbox evernote server.  Everything seems to work fine, except I can't see any of my notes.  I'm able to login from moodle, but all I see are notebooks.  I can actually create a new notebook and I see it, but I can't see any of the notes.

Any suggestions?  It sounds like a permissions issue.

Thanks, Scot

**Comment by FMCorz** [comment]
Thanks for the report, I'll have to have a look.

**Comment by FMCorz** [comment]
I have tried today and it worked for me. Are there attachments in your notes?

**Comment by jcmosaiclearning** [comment]
I am also running into this issue on a stock Moodle 3.2 instance with a full access API key on the evernote sandbox environment.

I can see my notebooks and tags on the sandbox instance, but the only actual *note* I can see is one that I added an attachment to(after reading your reply). The documentation is unclear, is that how this is supposed to work? Only attachments are available to be selected in moodle's upload dialog? 

Here is what I'm seeing in the response from repository_ajax.php when I browse into a notebook that I *know* has shared notes within it:

![evernote1](https://user-images.githubusercontent.com/32438971/31058071-7192c748-a6bb-11e7-84b1-6d7f329086be.PNG)

**Comment by FMCorz** [comment]
That is correct, only notes with attachments show up. I'm interested to hear what you intended to do with the other ones.

**Comment by jcmosaiclearning** [comment]
Okay, that was my misunderstanding then. I thought this repository would allow for users to pull in the contents of notes themselves.
